woocommerce/wc-admin
# Add loading indicators to the revenue report.

As a part of the open issues left for #316 and #225, this PR adds loading state to the revenue report. It follows the same pattern as the `TablePlaceholder` component by adding placeholder alternatives for `SummaryList` and `Chart`. It also adds the new inline documentation to `TablePlaceholder`.

<img width="986" alt="screen shot 2018-09-03 at 12 27 02 pm" src="https://user-images.githubusercontent.com/689165/44996274-f964cc80-af74-11e8-9be6-095e1651e171.png">

To Test:
* Visit the revenue report and confirm the loading indicators display correctly when a request is being made.
